topo/node/cptx/cptx.go

```diff
@@ -208,6 +208,36 @@ func (n *Node) ConfigPush(ctx context.Context, r io.Reader) error {
 	return nil
 }
 
+func (n *Node) ResetCfg(ctx context.Context) error {
+	log.Infof("%s resetting config", n.Name())
+
+	err := n.SpawnCLIConn(n.Namespace)
+	if err != nil {
+		return err
+	}
+
+	defer n.cliConn.Close()
+
+	cfgs := []string{
+		"load factory-default",
+		"delete system commit factory-settings",
+		"set system root-authentication encrypted-password $6$7uA5z8vs$cmHIvL0aLU4ioWAHPR0PLeU/mJj.JO/5pQVQoqRlInK3fJNTLYLhwiDi.Q6gHhltSB3S1P/.raEsuDSH7akcJ/",
```

Comment:
I'm a little conflicted on this, on one hand an encrypted password is best practice and on the other how will a user know the default password. In our example configs this is less of a problem because they are just examples. Users do not have to use them. Here we are setting up this encrypted password as part of kne_cli itself

I also doubt in this case the security concerns of simply using the unencrypted password. Maybe a middle ground is to comment the unencrypted password on this line so our users will still know the password without it appearing directly in the config itself?

Reply:
Sure, will do for arista

Reply:
I will add the comments for arista on a different PR after we hear back from them.